updatehandler/updatehandler.go

```diff
@@ -340,7 +340,9 @@ func (handler *Handler) init() {
 		})
 	}
 
-	doPriorityOperations(operations, false)
+	if err := doPriorityOperations(operations, false); err != nil {
```

Comment:
By design, this function never returns err in this case. Just void it:
```go
_ = doPriorityOperations
```
or change message to `Error init modules: %s`

Reply:
changed to _ = doPriorityOperations